cypress/e2e/awx/views/schedules.cy.ts

```diff
@@ -8,52 +8,55 @@ import { Schedule } from '../../../../frontend/awx/interfaces/Schedule';
-  function generateScheduleName(): string {
-    return `test-${testSignature}-schedule-${randomString(5, undefined, { isLowercase: true })}`;
-  }
+describe('Schedules - Create and Delete', () => {
+  describe('Schedules - Create schedule of resource type Job template', () => {
+    let organization: Organization;
+    let jobTemplate: JobTemplate;
+    let project: Project;
+    let inventory: Inventory;
 
-  before(function () {
-    project = this.globalProject as Project;
-    cy.createAwxOrganization().then((o) => {
-      organization = o;
-      cy.createAwxInventory({ organization: organization.id }).then((i) => {
-        inventory = i;
+    before(() => {
+      cy.createAwxOrganization().then((o) => {
+        organization = o;
+        cy.createAwxProject({ organization: organization.id }).then((proj) => {
+          project = proj;
+          cy.createAwxInventory({ organization: organization.id }).then((i) => {
+            inventory = i;
+            cy.createAwxJobTemplate({
+              name: 'E2E Credentials ' + randomString(4),
+              organization: organization.id,
+              project: project.id,
+              inventory: inventory.id,
+            }).then((jt) => {
+              jobTemplate = jt;
+            });
+          });
+        });
       });
     });
-  });
 
-  after(() => {
-    cy.deleteAwxInventory(inventory, { failOnStatusCode: false });
-    cy.deleteAwxOrganization(organization, { failOnStatusCode: false });
-  });
+    after(() => {
+      cy.deleteAwxJobTemplate(jobTemplate, { failOnStatusCode: false });
+      cy.deleteAwxInventory(inventory, { failOnStatusCode: false });
+      cy.deleteAwxProject(project, { failOnStatusCode: false });
 
-  it('can create a simple schedule, navigate to schedule details, then delete the schedule from the details page', () => {
-    cy.createAwxJobTemplate({
-      name: 'E2E Credentials ' + randomString(4),
-      organization: organization.id,
-      project: project.id,
-      inventory: inventory.id,
-    }).then((jt) => {
-      const scheduleName = 'E2E Simple Schedule' + randomString(4);
-      jobTemplate = jt;
+      cy.deleteAwxOrganization(organization, { failOnStatusCode: false });
+    });
+
+    it('can create a simple schedule or resource type Job template, navigate to schedule details, then delete the schedule from the details page', () => {
       cy.navigateTo('awx', 'schedules');
       cy.verifyPageTitle('Schedules');
+      const scheduleName = 'E2E Simple Schedule' + randomString(4);
       cy.getBy('[data-cy="create-schedule"]').click();
       cy.selectDropdownOptionByResourceName('schedule_type', 'Job template');
       cy.selectDropdownOptionByResourceName('job-template-select', `${jobTemplate.name}`);
-      cy.get('[data-cy="name"]').type(`${scheduleName}`);
+      cy.getByDataCy('name').type(`${scheduleName}`);
       cy.selectSingleSelectOption('[data-cy="timezone"]', 'Zulu');
```

Comment:
hey @acruzgon could we try to replace the`selectSingleSelectOption` with the `singleSelectByDataCy`?